runtime/compiler/env/JITaaSCHTable.cpp

```diff
@@ -221,6 +221,9 @@ bool JITaaSCHTableCommit(
       TR_MethodMetaData *metaData,
       CHTableCommitData &data)
    {
+   if (comp->fej9()->isAOT_DEPRECATED_DO_NOT_USE())
+      return true;
```

Comment:
Can we replace this with an assert now since we don't expect to call it in AOT mode?